src/openapi/schemas/id-string-params.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from "@hono/zod-openapi";
+
+const IdStringParamsSchema = z.object({
+  id: z.string().openapi({
```

Comment:
Path params will always be a string since they are in the URL 🤔.

What about `AlphaNumericIdParamsSchema` or `SlugParamsSchema` and include a regex that only allows letters, numbers, dashes and underscores?